src/util/loc.ts

```diff
@@ -73,7 +75,9 @@ export const loc = function (
       return 'L10N_ERROR[' + key + ']';
     }
   } catch (e) {
-    emitL10nError(key, bundleName, 'parameters');
+    if (!ignoreIncorrectParams) {
```

Comment:
do we have some info about what Error is thrown here? As in, we might want to emit the l10n error for all others errors we can catch here, with the exception of the Error about arguments missing